source/docs/articles/sites.md

```diff
@@ -1,33 +1,73 @@
+
+The **Visit Site** button is available for each environment so you can view the site in each environment. This helpful to view changes in Dev or Test before moving them to Live.
+
+## Team
+The Team tab allows you to change the site owner; add supporting agencies; and add, manage, and delete users. For more information, see [Team Management](/docs/articles/sites/team-management/).
+
+##Settings
+### Plan and Billing
+Here you can select or change your plan and payment method. For more information, see [Selecting a Plan](/docs/articles/sites/settings/selecting-a-plan/).
+
+### Add-Ons
+Pantheon offers several add-ons to help you build dynamic sites. These include [New Relic](docs/articles/sites/newrelic/new-relic-performance-analysis/), [Apache Solr](/docs/articles/sites/apache-solr/), and [Redis](/docs/articles/sites/redis-as-a-caching-backend/).
+
+###Delete Site
+If you no longer need your site, you can remove it here.
+**Note**: This is a permanent change and once you delete it your site cannot be restored.
+
+###About Site
+This tab shows general information about your site, such as the framework, upstream, date it was created, and your current plan.
+
+###PHP Version
+The site default PHP version is applied to every environment unless a PHP version is explicitly selected for each environment. Learn more about [Toggling Between PHP Versions](/docs/articles/sites/settings/toggling-between-php-versions/).
+
+## Support
+If you need assistance, our expert Customer Support team is here to help make building and managing your sites easier. You can contact Support by opening a ticket from your Dashboard. Learn more about our [scope of support](/docs/articles/scope-of-support/).
+
+##Public
+You can set a common password for accessing this environment to add an extra layer of security to prevent unwanted access to this environment. Learn more about [Locking Your Site](/docs/articles/sites/security/locking-your-site/).
+
 ## Connection Information
-## Delete Site
-## Deploy
-## Domains
-## Errors
-## Jobs Indicator
+Here you can find the [SSH clone URL](docs/articles/local/starting-with-git/), [Database Connection Information](/docs/articles/local/accessing-mysql-databases/), and [SFTP](/docs/articles/sites/code/developing-directly-with-sftp-mode/) connection information.
+
+## Code
+The core of the Pantheon Workflow is to move code up from Dev to Test to Live and content down from Live to Test to Dev.
+
+- **Code** includes plugins, modules, themes, CSS, JS—anything that's under Git version control.
+- **Content** includes files not under Git version control, like images and pdfs, and the database.  
+
+Learn how to use the [Pantheon workflow](/docs/articles/sites/code/using-the-pantheon-workflow/) to manage your site's code.
+
 ## Merge
-## MultiDev
-## PHP Version Toggle
-## Security
-## Settings
-## SSL
+Visit this tab when you have commits you need to merge into another environment.
```

Comment:
Tool only available on Dev. 

rephrase to indicate commits are coming from CDE's